src/containers/ListingPage/ListingPage.jsx

```diff
@@ -372,6 +372,10 @@ const ListingPage = ({ t, locale, location, isOembed }) => {
       : [];
   };
 
+  const getEmbedCode = (domain, filter) => {
+    return `<iframe src="${domain}/listing?filters[]=${filter}" frameborder="0" allowFullscreen="" />`;
```

Comment:
Her kan du bruke navnet på lista som tittel.
```suggestion
    return `<iframe aria-title="${filter}" src="${domain}/listing?filters[]=${filter}" frameborder="0" allowFullscreen="" />`;
```